src/System.Runtime.Caching/src/System/Runtime/Caching/Dbg.cs

```diff
@@ -0,0 +1,1068 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Now I see how elaborate this Dbg class is, it makes sense to keep all the Dbg.Assert's as is. (you do need to change the define DBG to DEBUG though)

Reply:
You can remove all these lines:
```
System.Runtime.Caching\src\System\Runtime\Caching\Dbg.cs:        [RegistryPermission(SecurityAction.Assert, Unrestricted=true)]
System.Runtime.Caching\src\System\Runtime\Caching\Dbg.cs:        [RegistryPermission(SecurityAction.Assert, Unrestricted=true)]
System.Runtime.Caching\src\System\Runtime\Caching\Dbg.cs:        [PermissionSet(SecurityAction.Assert, Unrestricted = true)]
System.Runtime.Caching\src\System\Runtime\Caching\Dbg.cs:        [ReflectionPermission(SecurityAction.Assert, Unrestricted=true)]
```